.readthedocs.yaml

```diff
@@ -0,0 +1,28 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+
+# Build documentation in the "doc/" directory with Sphinx
+sphinx:
+  configuration: doc/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+# formats:
+#    - pdf
```

Comment:
I think we can do also PDF. Readthedocs will display a download button, if I got it right.

Reply:
Yes, the issue is the pdf is too large to automatically build right now. Maybe the quota is increased with the paid version automatically, otherwise we can ask readthedocs to do it (see comment in [https://github.com/readthedocs/readthedocs.org/issues/8382](https://github.com/readthedocs/readthedocs.org/issues/8382)).

Reply:
I see

Reply:
Now that I had a closer look at the issue, I have had the problem myself early on when trying to build the documentation. I think that's the reason why I have been building the pdf on Windows. MikTeX does not seem to have the problem. I think the problem in the issue was the builder itself. We also switched to xelatex, probably partly because of the same problem.